net-misc/hylafaxplus/hylafaxplus-7.0.2.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit pam toolchain-funcs
+
+MY_PN="${PN/plus/}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Enterprise client-server fax package for class 1 and 2 fax modems"
+HOMEPAGE="http://hylafax.sourceforge.net"
+SRC_URI="mirror://sourceforge/hylafax/${MY_P}.tar.gz"
+
+SLOT="0"
+LICENSE="hylafaxplus"
+KEYWORDS="~amd64 ~x86"
+IUSE="jbig html lcms ldap mgetty pam"
+
+DEPEND="
+	>=sys-libs/zlib-1.1.4
+	app-text/ghostscript-gpl
+	virtual/mta
+	media-libs/tiff:0[jbig?]
+	virtual/jpeg:0
+	jbig? ( media-libs/jbigkit )
+	virtual/awk
+	lcms? ( media-libs/lcms )
+	ldap? (  net-nds/openldap )
+	pam? ( sys-libs/pam )
+	mgetty? ( net-dialup/mgetty[-fax] )
+	!net-dialup/mgetty[fax]
+"
+RDEPEND="${DEPEND}
+	net-mail/metamail
+	!net-dialup/sendpage
+"
+
+S="${WORKDIR}/${MY_P}"
+
+export CONFIG_PROTECT="${CONFIG_PROTECT} /var/spool/fax/etc /usr/lib/fax"
+export CONFIG_PROTECT_MASK="${CONFIG_PROTECT_MASK} /var/spool/fax/etc/xferfaxlog"
+
+PATCHES=(
+	"${FILESDIR}/ldconfig-patch"
+	"${FILESDIR}/${PN}-CVE-2020-1539x.patch"
+	)
+
+src_prepare() {
+	default
+
+	# force it not to strip binaries
+	for dir in etc util faxalter faxcover faxd faxmail faxrm faxstat \
+		hfaxd sendfax sendpage ; do
+			sed -i -e "s:-idb:-idb \"nostrip\" -idb:g" \
+				"${dir}"/Makefile.in || die "sed failed"
```

Comment:
Would be better to see in `sed` error message which directory it failed on. For debugging.